### ISSUE TITLE
False positive of `no-import-module-exports` on `module` property imported from package

### ISSUE THREAD

**Issue by BarryThePenguin** [import-js/eslint-plugin-import#2314]
Similar to #2181, I'm seeing a false positive when using `qunit` with ember.

`qunit` exports a `module` property that allows tests to be grouped under a module. It also has `module.only()`, `module.skip()`, and `module.todo()` methods that apply behaviour to all tests in that module.

```js
import { module, test } from 'qunit'

module.skip('A module to test', function () {
  test('A test', function () {
    ...
  })
})
```

The `no-import-modules-exports` is reporting these methods as errors.

**Comment by ljharb** [comment]
Fixed by #2315.
